### ISSUE TITLE
Old Hillsbrad Foothills

### ISSUE THREAD

**Issue by Tabascos** [ccshiro/cc-buglist#2391]
**Hey there!

We have done an OHF heroic run recently, and I found a bug there. When you enter the Keep, and cross the first bridge, then turn to the right, there will be wooden stairs (the ones Leutenant uses to get down). If you stand on top of them, and pull the pack of mobs infront of you (straight infront, at the barrack door) to your spot on the stairs, they wont kill anybody apart of the target who pulled. I mean, if you pull them and your tank (or the person who pulled) dies being on top of the stairs, they will just evade and rest of your group will survive. I have tried it about 7 times, and bug occured 3 times. **

**Comment by ccshiro** [comment]
Not able to reproduce. Do you guys have a recording or some sense of what went different the tries it happened?

**Comment by Tabascos** [comment]
Nope, unfortunately not :( We were 5 people, tank was standing on the stairs, rest were behind, at the grass.
